docs/design.html

```diff
@@ -124,6 +124,7 @@ <h3 class="anchor-heading"><a id="maximizingefficiency" class="anchor-link"></a>
     and copied out to user-space every time it is read. This allows messages to be consumed at a rate that approaches the limit of the network connection.
     <p>
     This combination of pagecache and sendfile means that on a Kafka cluster where the consumers are mostly caught up you will see no read activity on the disks whatsoever as they will be serving data entirely from cache.
+    When the transport layer uses the SSL protocol, sendfile will not be used due to the need to encrypt the data read.
```

Comment:
Nice addition and clarification here! Adding a few suggestion below for your consideration:

1. It might be better if we could add the configuration name here which enables SSL protocol. This clarification will help the users understand what configuration could have an impact on zero copy performance.

2.  This line is an important callout. Perhaps start with a new paragraph so that it jumps out.

3. Perhaps add the reasoning for what prevents usage of `sendfile` with SSL such as, 
> "TLS/SSL libraries operate at the user space (in-kernel TLS is currently not supported by Kafka). Due to this restriction, `sendfile` API could not be used when transport layer uses SSL protocol".